src/ml_gym/frontend/dashboard/src/app/store.tsx

```diff
@@ -2,13 +2,15 @@ import { configureStore, ThunkAction, Action, } from '@reduxjs/toolkit'
 import jobStatusReducer from '../features/jobsStatus/jobsStatusSlice'
 import modelStatusReducer from '../features/modelsStatus/modelsStatusSlice'
 import modelEvaluationReducer from '../features/modelEvaluations/modelEvaluationsSlice'
+import experimentConfigReducer from '../features/experimentConfig/experimentConfigSlice';
 
 
 export const store = configureStore({
   reducer: {
     jobsStatus: jobStatusReducer,
     modelsStatus: modelStatusReducer,
-    modelsEvaluation: modelEvaluationReducer
+    modelsEvaluation: modelEvaluationReducer,
+    experimentConfig: experimentConfigReducer
```

Comment:
Codacy has a fix for the issue: [Insert `,`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=10020887)
```suggestion
    experimentConfig: experimentConfigReducer,
```